### DISCUSSION TITLE
Show lookup values in content view

### DISCUSSION THREAD

**Discussion by ammaraga** [directus/directus#12760] [Archive]
I have a "projects" and "teams" collection. "projects" has a "team_id" field which is a key to "teams" table . I need to be able to see "team_name" (a field in "teams" collection) in content view. The relationship exists and I see the looked up value when I open the project, but I don't see it in content (list) view. Any way to have it displayed there?

**Comment by rijkvanzanten** [comment]
You can select nested fields on the tabular layout:

<img width="1583" alt="CleanShot 2022-04-14 at 09 57 02@2x" src="https://user-images.githubusercontent.com/9141017/163406154-a371c630-126d-429b-b3ef-0e3ae9e7004f.png">

**Reply by ammaraga** [reply]
How do I get to "nested fileds"? I don't see the "+" you have in your screenshot?
![image](https://user-images.githubusercontent.com/74614063/163410184-29e291e8-e6ed-4894-ae73-8dd9a61f66cd.png)

**Reply by benhaynes** [reply]
Are you on the latest version of Directus?

**Reply by ammaraga** [reply]
Directus 9.0.0

**Reply by rijkvanzanten** [reply]
> Directus 9.0.0

You're pretty far behind then! I'd strongly recommend upgrading to the latest version, and staying up to date 👍🏻

**Reply by ammaraga** [reply]
OK, I'm running 9.9.0 now, and I have the "+", but it doesn't show "nested fields", just hte ones belonging to the table. There is a foreign key between the 2 tables, but no fields from the related tables are shown. Am I missing something?

EDIT: ah, it seems to be working now. Under Settings/DataModel/projects/team_id/Display: "Related Values" and "Team Name"
